services/ui-src/src/shared/types/MeasureTemplate.tsx

```diff
@@ -29,5 +29,6 @@ export interface MeasureTemplateData {
   custom?: customData;
   opm?: {
     excludeOptions?: string[];
+    componentFlag?: string;
```

Comment:
Let's use the existing type definition that restricts this to `"DEFAULT" | "AIF" | "IU" | "PCR"`

```suggestion
    componentFlag?: ComponentFlagType;
```

Reply:
going to do this as a tag along in a separate PR since it's small